config/routes.rb

```diff
@@ -14,8 +14,9 @@
     end
 
     namespace :api do
-      resources :tasks, :only => [:show] do
+      resources :tasks, :only => [:show, :index] do
         post :bulk_search, :on => :collection
+        put :bulk_resume, :on => :collection
```

Comment:
I don't see this actions as standard REST calls, but rather RPCs  in this case, so I would not be that strict about the this division: simplifying that and using POST for this would work for me a bit better, but not a big deal.